guide/.gitignore

```diff
@@ -1 +1,3 @@
 book
+build/*
+book.toml
```

Comment:
I'm not sure if removing `book.toml` from source is a good idea, but maybe there's no other clean way if this is now generated by a script...